Documentation/network/servicemesh/gateway-api/gateway-api.rst

```diff
@@ -0,0 +1,36 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gs_gateway_api:
+
+*******************
+Gateway API Support
+*******************
+
+Cilium supports Gateway API v0.5.1 for below resources, all the upstream conformance tests
+are passed.
```

Comment:
Nit: Maybe clarify this, too?

```suggestion
Cilium supports Gateway API v0.5.1 for the resources listed below, and passes `all upstream conformance tests <add_a_link_here>`__.
```

(Or `all related upstream conformance tests` if they're just the tests relating to supported resources?)

Reply:
I think that `all the Core conformance tests, plus the ReferenceGrant extended tests` is the most correct. (This should change in the next release, when ReferenceGrant is included in the core suite).